saleor/app/app_manifest_sample.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "saleor.app.sample",
+  "version": "1.0.0",
+  "name": "My Wonderful App",
+  "about": "My Wonderful App is a wonderful App for Saleor.",
```

Comment:
This file will be used as official template for app manifests, are you sure we want to have it?

Reply:
We have the same sample in docs. What is wrong with this?